config.toml

```
@@ -153,17 +149,29 @@ theme = "elate"
       alt = "Person2"
 
     [[params.testimonials.item]]
-      quote = "&ldquo;J’avais besoin de reprendre confiance en moi dans le cadre d’une négociation. Ses grandes qualités d’écoute et d’analyse m'ont aidé à mettre au point une stratégie pour gérer ma situation de manière optimale.&rdquo;"
+      quote = "&ldquo;J’avais besoin de reprendre confiance en moi dans le cadre d’une négociation. Ses grandes qualités d’écoute et d’analyse m'a aidé à mettre au point, au fil des séances de coaching, une stratégie pour gérer ma situation de manière optimale.&rdquo;"
       person = 'Grégoire Hallé, Conservateur du Patrimoine, Musée des Beaux-Arts de Draguignan'
       img = "person3.jpg"
       alt = "Person3"
```

L'aternative textuelle de l'image peut être renseignée :

```suggestion
      alt = "Grégoire Hailé"
```